<launch>
    <arg name='map_name' default='Town03'/>

    <param name="/t4ac/mapping/maps/xodr" type="string" value="/workspace/team_code/catkin_ws/src/t4ac_mapping_layer/maps/xodr/"/>

    <param name="t4ac/map_name" type="string" value="$(arg map_name)" />
    <node pkg="t4ac_map_monitor_ros" type="map_visualizator.py" name="map_visualizator_node" output="screen"/>
    <node pkg="t4ac_map_monitor_ros" type="map_monitor.py" name="map_monitor_node" output="screen"/>
    <node pkg="t4ac_map_monitor_ros" type="monitor_visualizator.py" name="monitor_visualizator_node" output="screen"/>
</launch>